variables.tf

```diff
@@ -84,6 +84,12 @@ variable "github_admins" {
   description = "A list of Github teams that should have admins access"
 }
 
+variable "github_organization" {
+  type        = string
+  description = "The Github organization to connect the workspace to"
+  default     = null
+}
```

Comment:
Order of fields is `type`, `default`, `description`:

Can you update the variable so that it's inline with other variable defs:

```hcl
variable "github_organization" {
  type        = string
  default     = null
  description = "The Github organization to connect the workspace to"
}
```